managers/s3.py

```diff
@@ -37,11 +42,49 @@ def createS3Bucket(self, bucketName, bucketPermissions):
         raise S3Error('Unable to create bucket in s3')
 
 
-    def createManifestInS3(self, manifestPath, manifestJSON, s3_bucket: str):
+    def store_pdf_manifest(self, record, bucket_name, flags=None, requires_login: bool=False):
```

Comment:
let's add some types to the params to make sure they are clear. 

Instead of the flags defaulting to None, we can default to FileFlags(reader=True)